From eb0fe1faac5d3816442dea578167442730f0be34 Mon Sep 17 00:00:00 2001
From: Thyrus11 <thyrus11@gmail.com>
Date: Sat, 21 Feb 2015 10:41:01 +0100
Subject: [PATCH] Themes: Make dialpad seperator line theme-able

Using an existing value in colors is defined
(dialpad_seperator_line_color) in AOSP, but overwritten in layout

Changed the original commit's color name due to layout was #e3e3e3 and
dialpad_seperator_line_color is #dadada

Change-Id: Ia4b610185bac6c34e0f6bd1b00a6f0c0480d5ee0
---
 res/layout/dialpad_view_unthemed.xml |  4 ++--
 res/values/projekt_colors.xml        | 20 ++++++++++++++++++++
 2 files changed, 22 insertions(+), 2 deletions(-)
 create mode 100644 res/values/projekt_colors.xml

diff --git a/res/layout/dialpad_view_unthemed.xml b/res/layout/dialpad_view_unthemed.xml
index 7bc4536..3fcf0da 100644
--- a/res/layout/dialpad_view_unthemed.xml
+++ b/res/layout/dialpad_view_unthemed.xml
@@ -56,7 +56,7 @@
         <View
             android:layout_width="match_parent"
             android:layout_height="1dp"
-            android:background="#e3e3e3" />
+            android:background="@color/dialpad_separator_color" />
 
     </LinearLayout>
 
@@ -148,7 +148,7 @@
     <View
         android:layout_width="match_parent"
         android:layout_height="1dp"
-        android:background="#e3e3e3" />
+        android:background="@color/dialpad_separator_color" />
 
     <Space
         android:layout_width="match_parent"
diff --git a/res/values/projekt_colors.xml b/res/values/projekt_colors.xml
new file mode 100644
index 0000000..163e4f0
--- /dev/null
+++ b/res/values/projekt_colors.xml
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Copyright (c) 2016 Projekt Substratum
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+
+<resources>
+    <color name="dialpad_separator_color">#e3e3e3</color>
+</resources>
-- 
2.7.4

